src/main/java/io/voucherify/client/model/order/response/OrderResponse.java

```diff
@@ -35,6 +35,9 @@ public class OrderResponse {
 
   private CustomerResponse customer;
```

Comment:
I noticed that OrderResponse in case of redemption has also `redemptions` object